casr/src/util.rs

```diff
@@ -17,17 +17,18 @@ use std::path::{Path, PathBuf};
 use std::process::{Command, Stdio};
 use std::sync::RwLock;
 
-/// Call casr-san with the provided options
+/// Call sub tool with the provided options
 ///
 /// # Arguments
 ///
 /// * `matches` - casr options
 ///
-/// * `tool` - tool, that called casr-san
+/// * `tool` - tool, that called sub tool
 ///
 /// * `argv` - executable file options
-pub fn call_casr_san(matches: &ArgMatches, argv: &[&str], tool: &str) -> Result<()> {
-    let mut cmd = Command::new("casr-san");
+pub fn call_sub_tool(matches: &ArgMatches, argv: &[&str], tool: &str) -> Result<()> {
```

Comment:
we'd better pass sub tool in `tool` argument

Reply:
Done